sycl/test/basic_tests/accessor/accessor_default_ctor.cpp

```diff
@@ -11,9 +11,7 @@ int main() {
   assert(B.byte_size() == 0);
   // The return values of get_pointer() and get_multi_ptr() are unspecified.
   assert(B.get_pointer() == nullptr);
-  // TODO: uncomment check with get_multi_ptr() when SYCL 2020 mupti_ptr feature
-  // will be merged
-  // assert(B.get_multi_ptr() == nullptr);
+  assert(B.get_multi_ptr<sycl::access::decorated::yes>() == nullptr);
```

Comment:
For good measure, could you please add a `sycl::access:decorated::no` and `sycl::access::decorated::legacy` test case here?

Reply:
thanks @steffenlarsen , done.